\documentclass[12pt]{article}
\usepackage{amsmath}
\usepackage{graphicx,psfrag,epsf}
\usepackage{enumerate}
\usepackage{natbib}

\newcommand{\blind}{0}

\addtolength{\oddsidemargin}{-.75in}%
\addtolength{\evensidemargin}{-.75in}%
\addtolength{\textwidth}{1.5in}%
\addtolength{\textheight}{1.3in}%
\addtolength{\topmargin}{-.8in}%


\begin{document}


%\bibliographystyle{natbib}

\def\spacingset#1{\renewcommand{\baselinestretch}%
{#1}\small\normalsize} \spacingset{1}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\if0\blind
{
  \title{\bf High-latitude seasonal dynamics in fish diversity as evident by eDNA metabarcoding.}
  \author{Jon-Ivar Westgaard\\
    Institute of Marine Research\\
    and \\
    Gledis Guri\\
    Institute of Marine Research}
  \maketitle
} \fi

\if1\blind
{
  \bigskip
  \bigskip
  \bigskip
  \begin{center}
    {\LARGE\bf Title}
\end{center}
  \medskip
} \fi

\bigskip
\begin{abstract}
Abstrct.
\end{abstract}

\noindent%
{\it Keywords:}  Fish, 

\spacingset{1.45}
\section{Introduction}
\label{sec:intro}

Seasonal variations, that is changes in light, temperature, currents etc., play a crucial role in shaping marine ecosystems, influencing everything from the distribution of species to nutrient and carbon cycling. This is especially pronounced in high-latitude systems where differences between seasons are stronger with a short and intense productive season. As of this, sub-arctic fjords are characterized by strong seasonal shifts in environmental variables such as temperature, salinity, and light regime. This in turn affects the annual cycle of the primary production, a key stone in ecosystems (Michelsen et al., 2017; Visser et al., 2020). Subsequently, all these factors give cues for reproduction, migrations, abundance, and distribution of species. Understanding ocean seasonality is essential for managing marine resources and predicting the impact of climate change on marine ecosystems.
There are several factors influencing the diversity and quantity of fish species found in sub-arctic fjords. The two main factors are food availability and reproduction (REF). Both these factors are seasonal dependent and are reflected in feeding and spawning migrations.
The majority of fish species have an annual reproduction cycle after maturation (Folkvord et al., 2014). In sub-arctic marine ecosystems reproduction usually takes place in spring, often accompanied with large scale spawning migrations (Gjosæter, 1998; Dalpadado et al., 2000). Which is reflected in the increased biomass of fish species entering the fjord systems. This timing of reproduction can be seen in light of the “match-mismatch” context (Cushing, 1990) where food availability is crucial when the newly hatched fish larvae has spent its energy reserve in the yolk sac and starts external feeding. These larvae feed mostly on zooplankton, which is linked to phytoplankton, in which blooms are conditioned on light and nutrient availability, initiated by the cascading effects that occur in this season (Visser et al., 2020). This balance between prey availability and predator demands is delicate as it involves several food web links. Facing global warming, the rise in ocean temperatures can disrupt this balance by introducing trophic mismatches if the timing of multiple trophic levels responds to climate change in different ways (Sydeman and Bograd, 2009).
The fish communities of sub-arctic fjords are often characterized by both stationary (coastal) and migratory (offshore) species and some species comprise both strategies (i.e., Atlantic cod, herring, saithe etc.). The most common seasonal pattern for species that have both a stationary and migratory component is that migratory components enter fjord systems to spawn alongside the resident component. Alternatively, migratory species also conduct these spawning migrations and enter fjords where the species does not exist in other seasons. For the latter case, this will temporally alter the species composition in the fjords. The former case will not have this effect but will be reflected in an increased biomass of the species. By understanding how fish assemblages vary across seasons in fjords an increased knowledge of species seasonal dynamics and interactions can be attained.   

An effective approach in detecting and monitoring of species diversity is through metabarcoding of environmental DNA (eDNA). With emerging technologies massive parallel DNA sequencing of environmental samples (e.g., water, soil, and sediments) has shown itself to have a wide range of applications, both in species detection, and in recent years utilizing its quantitative properties in estimating species density. Its robustness has been demonstrated in several studies in various ecosystems (REF). The non-invasive nature of sampling eDNA means that it has negligible disturbance on the studied ecosystem. In addition, analysis of eDNA is less dependent on taxonomic expertise, and is cost-effective (REF).
In this study we use metabarcoding of water derived eDNA to study variability of fish species composition and abundance among four sub-arctic fjords in northern Norway over three seasons, spring, summer, and autumn. 


\section{Methods}
\label{sec:meth}
Sampling
A total of 25 localities in four different sub-arctic fjords in northern Norway were sampled in three seasons (spring, summer and autumn) from 2018 to 2021 (Figure 1), summing up to xxx samples. At each locality, 2 liters of sea water were sampled in triplicate for three depths and subsequently filtered through a 0.22$\mu$M Sterivex filter (Merck-Millipore). The filters were stored at -20$^\circ$C immediately after filtration and transferred to -80$^\circ$C when the research vessel docked. For further sampling details see Guri et al. 2023. The four fjords sampled in this study is representative of the heterogeneity of fjords in northern Norway. From the long and sheltered Balsfjord to different sized Olderfjord, Frakkfjord and Bergsfjord, which is more exposed to the open ocean. The number of sampled localities in each fjord is correlated with the size of the fjord (Figure 1).


Contamination control
To monitor possible contamination, negative controls were included in each step of the workflow. This included two negative controls per locality during sampling. One water (MilliQ water) and one air control. In the DNA extraction one negative per block of 12 samples were included.

Library preparation
DNA was extracted from the filters using DNeasy PowerWater Sterivex Kit (Qiagen GmbH) as described in Guri et al. (2023). The amplification of metabarcodes was done using the MiFish 12S assay (Miya et al., 2015). Allowing for a one-step PCR protocol, fusion primers were applied. That is primers with instrument specific adaptors and indexes attached to the template specific sequence. All PCR’s were done in triplicate to reduce the amplification stochasticity in the reactions. Verification of positive amplification and correct product size was done using the capillary based Qiaxel (Qiagen GmbH). The individual PCR products were subsequently pooled into nine libraries and thereafter size selected by electrophoresis on a 2\% agarose gel and excising the PCR product of interest (~300 bp). Purification of the excised products were done using the GeneJet Gel Extraction and DNA cleanup Micro Kit (Thermo Fisher Scientific). DNA concentrations of the purified sequencing libraries were measured with the Qubit dsDNA HS kit (Thermo Fisher Scientific) and diluted to 50 µM and spiked with 4 µL of Ion Calibration Standard and subsequently loaded onto the Ion Chef instrument (Thermo Fisher Scientific). Libraries were sequenced on a GeneStudio S5 with the Ion 530 (4 libraries) or 540 chip (5 libraries) and the 200 bp protocol.

Bioinformatics
Demultiplexing of the sequences were automatically done by the Torrent Suite™, inbuilt in the sequencer, with the default settings. Chimeric sequences were filtered out using the uchime-denovo algorithm in VSEARCH (Rognes et al., 2016). Clustering the sequences into MOTU’s (Molecular Operational Taxonomic Units) was done with SWARM v2 (Mahe et al., 2015), with a distance of d = 3. Singletons were removed before the taxonomic assignment of the sequences through the ecotag  function in OBITools (Boyer et al., 2016). The taxonomically assigned MOTU’s with a higher number of reads than 10 were BLAST searched against the NCBI nucleotide (nt) database (accession date?) with the BLASTn algorithm. An E-value of 1e-30 and a percentage identity of 90 were defined as thresholds. Taxa that were not assigned to fish, Actinopterygii or Chondrichthyes, were removed from the dataset along with MOTU’s with a read abundance higher than 10% in negative controls over the environmental samples.


\section{Verifications}
\label{sec:verify}

\section{Conclusion}
\label{sec:conc}


\bigskip
\begin{center}
{\large\bf SUPPLEMENTAL MATERIALS}
\end{center}

\begin{description}

\item[Title:] Brief description. (file type)

\item[R-package for  MYNEW routine:] R-package ÒMYNEWÓ containing code to perform the diagnostic methods described in the article. The package also contains all datasets used as examples in the article. (GNU zipped tar file)

\item[HIV data set:] Data set used in the illustration of MYNEW method in Section~ 3.2. (.txt file)

\end{description}

\begin{thebibliography}{}

\bibitem[Azzalini(2005)]{azza:05}
Azzalini, A. (2005).
\newblock The skew-normal distribution and related multivariate families.
\newblock \emph{Scandinavian Journal of Statistics} \textbf{32}, 159--188.

\end{thebibliography}{}

\end{document} 